main.go

```diff
@@ -629,6 +629,20 @@ func main() {
 	if err != nil {
 		l.Panicf("Server problem: %+v", err)
 	}
+	updateSignal := make(chan os.Signal, 1)
+	signal.Notify(updateSignal, unix.SIGHUP)
```

Comment:
We have another signal handler somewhere in that file. Please keep them close to each other. If we can't move that one, let's move the other one there